packages/auth/src/AuthCard/AuthCard.tsx

```diff
@@ -33,66 +33,35 @@ export const AuthCard = ({
   const { logo } = React.useContext(LogoContext);
 
   return (
-    <Card
-      variant="primary"
-      sx={{
-        maxWidth: '340px',
-        fontFamily: 'body',
-        padding: 7,
-        paddingTop: 8,
-        border: 'none',
-        background: 'white',
-      }}
-    >
+    <Card variant="forms.auth.cardContainer">
```

Comment:
We can't use a specific variant for `auth` because the card has a role global. Every `primary` card should have the same style. For instance, if we want to change the primary card, we'd have to change `forms.auth.cardCointainer` and `card.primary`. This is bad. `card.primary` should affect the whole application.

Remove all `.auth.` of all variants.